### DISCUSSION TITLE
Payload returns as application/text by default, changing it application/json  payload  is return as string

### DISCUSSION THREAD

**Discussion by jozcar** [FirelyTeam/firely-net-sdk#2487] [General]
We need the GET request to return the payload by default as application/JSON instead of application/text. Currently, it is returning as application/text. We are utilizing the Bundle Class to bundle multiple Resource records. To ensure the controller returns application/JSON, I tried using bundle.ToJSON(), but it returns the payload as a String with escaped content. On the other hand, returning just the bundle without .ToJSON() returns it as application/JSON, but the 'resourceType' element is missing from the payload.   The documentation does state the .ToJSON() does return as String, any guidance is welcome

**Comment by ewoutkramer** [comment]
You will need the SDK's serializers, so, yes, you will need to use `ToJson()`, which will serialize to a string (including escaped content). Only these serializers will produce valid FHIR Json. 

However, what MIME type you are using does not depend on the serializer, but on the way you configure your webservice, so you have full control over that yourself.  Take a look at https://github.com/FirelyTeam/spark to get some inspiration on how to configure .NET webservices to get this done!
